src/typingsInstaller/nodeTypingsInstaller.ts

```diff
@@ -13,15 +11,25 @@ namespace ts.server.typingsInstaller {
 
     class FileLog implements Log {
         private logEnabled = true;
-        constructor(private readonly logFile?: string) {
+        private readonly logFile: string = '';

```

Comment:
I think it would be better if `logEnabled` was just rolled into the optionality of `logFile` -- if the log is enabled `logFile` should be set, otherwise `logFile` will be undefined. Then we don't have to initialize `logFile` with a value it should never actually use.